APIhomework/src/index.js

```diff
@@ -3,19 +3,23 @@ function getCurrentPosition(position) {
   let lat = position.coords.latitude;
   let lon = position.coords.longitude;
   let url = `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&units=imperial&appid=${apiKey}`;
-  axios.get(url).then(showTemperature);
+  axios.get(url).then(function(response) {
+    showTemperature(response.data.main.temp, response.data.name, response.data.sys.country);
+  });
 }
-
-function showTemperature(response) {
-  let temperature = Math.round(response.data.main.temp);
-  let currentTemp = document.querySelector(".currently-temp");
-  currentTemp.innerHTML = `${temperature}°F`;
-  let currentCity = document.querySelector(".currently-city");
-  let currentCityName = response.data.name;
-  currentCity.innerHTML = `${currentCityName}`;
-  let country = document.querySelector(".currently-country");
-  let currentCountryName = response.data.sys.country;
-  country.innerHTML = `${currentCountryName}`;
+                      
+function showTemperature(temp, country, city = null) {
+  const temperature = Math.round(temp);
+  const city = city.trim();
+  const country = country.trim();
+    
```

Comment:
I like to group variable definitions together. So all the value variables are declared, then all the DOM elements are declare. To some degree this is simply personal preference but I believe it makes code more readable / well organized.

Also, I sometimes suffix variable names to help give hints to the person reading the code as to what the variable is. For example, I often suffix my HTML elements with "El" so you can infer that the variable is an HTML element.